src/virtualdom/Fragment/prototype/rebind.js

```diff
@@ -3,7 +3,9 @@ import { assignNewKeypath } from 'shared/keypaths';
 export default function Fragment$rebind ( oldKeypath, newKeypath ) {
 
 	// assign new context keypath if needed
-	assignNewKeypath( this, 'context', oldKeypath, newKeypath );
+	if ( !this.owner || this.owner.hasContext ) {
+		assignNewKeypath( this, 'context', oldKeypath, newKeypath );
+	}
```

Comment:
Could we just do this?

```js
if ( this.context ) {
  assignNewKeypath(...);
}
```

Or would that cause other problems? That way we wouldn't need `hasContext`.

Or maybe [this line](https://github.com/ractivejs/ractive/blob/dev/src/shared/keypaths.js#L71) should actually be

```js
if ( !existingKeypath || ( existingKeypath.equalsOrStartsWith( newKeypath ) || !existingKeypath.equalsOrStartsWith( oldKeypath ) ) ) {
  return;
}
```